### PR TITLE
chore(egen): Follows new template, removes IS_TESTING, REGION --> LOCATION

### PR DESCRIPTION
**Pull request by krishr2d2**
<br>
This PR includes the following updates to the `automl_tabular_regression_model_evaluation.ipynb` notebook:

1. Replace REGION with LOCATION.
2. Remove IS_TESTING condtion in the Cleaning up section.
3. Adds collab enterprise logo.
4. Made structural changes according to new notebook template.
5. Contraction of words (do not --> don't).
6. Some minor heading/sentence corrections.
<br><br><br>

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

<br>

### CODE REVIEW

File: notebooks/official/model_evaluation/automl_tabular_regression_model_evaluation.ipynb [hunk 1]
```diff
@@ -776,9 +747,9 @@
         "id": "U2zocUvk2YVs"
       },
       "source": [
-        "In the results from last step, click on the generated link to see your run in the Cloud Console.\n",
+        "In the results from last step, click on the generated link to see your run details in the Cloud Console.\n",
```
Comment (katiemn) [inline notebooks/official/model_evaluation/automl_tabular_regression_model_evaluation.ipynb:750]: Cloud Console -> Google Cloud console

File: notebooks/official/model_evaluation/automl_tabular_regression_model_evaluation.ipynb [hunk 2]
```diff
@@ -32,24 +32,26 @@
         "# Vertex AI Pipelines: Evaluating batch prediction results from AutoML Tabular regression model\n",
```
Comment (kittyabs) [inline notebooks/official/model_evaluation/automl_tabular_regression_model_evaluation.ipynb:32]: lower case "tabular": AutoML tabular

File: notebooks/official/model_evaluation/automl_tabular_regression_model_evaluation.ipynb [hunk 3]
```diff
@@ -76,7 +78,7 @@
         "\n",
         "In this tutorial, you learn how to evaluate a Vertex AI model resource through a Vertex AI pipeline job using `google_cloud_pipeline_components`:\n",
         "\n",
-        "This tutorial uses the following Google Cloud ML services and resources:\n",
+        "This tutorial uses the following Vertex AI services:\n",
         "\n",
         "- Vertex AI Datasets (Tabular)\n",
```
Comment (kittyabs) [inline notebooks/official/model_evaluation/automl_tabular_regression_model_evaluation.ipynb:83]: change to: Vertex AI datasets (tabular)

File: notebooks/official/model_evaluation/automl_tabular_regression_model_evaluation.ipynb [hunk 4]
```diff
@@ -76,7 +78,7 @@
...
         "- Vertex AI Training (AutoML Tabular Training)\n",
```
Comment (kittyabs) [inline notebooks/official/model_evaluation/automl_tabular_regression_model_evaluation.ipynb:84]: change to: Vertex AI Training (AutoML tabular training)

**Review by kittyabs — COMMENTED** [review]
numerous edits required.

**Review by katiemn — APPROVED** [review]

**Pull request merged**